jena-base/src/main/java/org/apache/jena/atlas/lib/Timer.java

```diff
@@ -69,4 +69,13 @@ static public String timeStr(long timeInterval) {
     protected String timeStr(long timePoint, long startTimePoint) {
         return timeStr(timePoint - startTimePoint) ;
     }
+
+    /** Time an operation. Return the elapsed time in milliseconds. */
+    public static long time(Runnable action) {
```

Comment:
There's an implied assumption here that any exceptions thrown by the runnable are handled by the caller